packages/language-support/src/tests/consoleCommands.test.ts

```diff
@@ -456,7 +475,7 @@ describe('parameters', () => {
         length: 1,
         position: { line: 0, startCharacter: 7, startOffset: 7 },
         token: 'x',
-        tokenType: 'variable',
```

Comment:
This is not correct, it should still be a variable

Reply:
After shifting the definition of the lambda expression in the parser to use parameterName, this shifted to be symbolicName. I figured that was just an effect of shifting to parsing as a different token, but maybe not? Having a bit of trouble finding where it's assigned..